freetextresponse/private/view.less

```diff
@@ -66,11 +74,17 @@
     display: inline-block;
   }
 
-  .word-count-error {
-    color: #A50000;
+  //  DEBUG - should have access to built-int
+  //  capa_alert in need in lms and studio
```

Comment:
They don't exist in the workbench but do exist in LMS.  So to view the functionality in workbench I kept the css classes.  "should have access to capa_alert class in lms and studio"